package.json

```
@@ -2,7 +2,6 @@
   "name": "agent-builder",
   "version": "1.0.0",
   "main": "eslint.config.js",
-  "type": "module",
   "scripts": {
     "start:vercel-agent": "npx cross-env NODE_ENV=production node -r dotenv/config src/vercel/index.ts",
```

There is no build command, leading to errors when trying to run the scripts.
The option `--noImplicitAny false` is also required to avoid any compilation errors. I couldn't find another way.
```suggestion
  "scripts": {
    "build": "tsc --noImplicitAny false",
    "start:vercel-agent": "npx cross-env NODE_ENV=production node -r dotenv/config src/vercel/index.ts",
```